docs/data/date-pickers/accessibility/accessibility.md

```diff
@@ -0,0 +1,71 @@
+---
+productId: x-date-pickers
+title: Date and Time Pickers - Date format and localization
```

Comment:
Did you intend to add this page on this PR? 🤔 
If so, then this title should better reflect the page content and maybe we'd need to adjust the PR title. 😉

Reply:
Looks like I did something wrong 🙈